gradle-plugin/src/main/kotlin/net/weavemc/gradle/configuration/DependencyManager.kt

```diff
@@ -35,10 +36,7 @@ private fun Project.addMinecraftAssets(version: MinecraftVersion) {
     val client = versionInfo.downloads.client
     DownloadUtil.checksumAndDownload(URL(client.url), client.sha1, version.minecraftJarCache.toPath())
 
-    repositories.maven {
-        name = "mojang"
-        setUrl("https://libraries.minecraft.net/")
-    }
+    repositories.maven("https://libraries.minecraft.net/")
```

Comment:
The Kotlin DSL simply refused to work here, so this was my solution, if you're able to make it work lmk

Reply:
This is fine.